Generate and add sleeping evs icon

PetroSilenius opened the pull request:
## Background

When showing EVS scores you sometimes have a situation where you don't have any answers yet. There was no icon to describe that scenario as all current EVS icons are related to existing scores

[What is EVS (Experience Value Score)?](https://www.lyyti.com/en/evs)

## 😴 New EVSSleeping icon

- Adds new EVSSleeping icon to describe no answers scenario

kacperkosinski reviewed (approved):
Great job!

Would be nice if you would share how you created this base on figma sketch. I was trying it myself by manually changing elements inside SVG, but failed 🥲

PetroSilenius commented:
> Great job!
> 
> Would be nice if you would share how you created this base on figma sketch. I was trying it myself by manually changing elements inside SVG, but failed 🥲

Sure, I'll send a video in Slack! Here's the steps I did for anyone outside Lyyti interested
- Copied icon in Figma, it had some text elements + vectors
- I flattened the text elements in Figma to vectors
- I turned the group to a component (not necessary, but easier to deal with it in Figma in the future)
- Resized the component to 24x24 since that's the default viewbox of a MUI SVGIcon
- I copied the component as an SVG
- Pasted the svg content into an svg file on my machine
- Ran `npm run generate-icon --path=PATH_TO_SVG_FILE`
- Removed the fill colors from path and circle elements in the `src/icons/EVSSleeping.tsx` component
- Renamed the component
